src/pages/besquare/components/sections/_what-is_besquare.js

```diff
@@ -38,10 +38,15 @@ const WhatIsBeSquare = () => {
         <Section>
             <ContentContainer>
                 <ContentWrapper>
-                    <Title as="h2" text_align={'left'} max_width={['450px']} margin={['0', '0 auto']}>
+                    <Title
+                        as="h2"
+                        text_align={'left'}
+                        max_width={['450px']}
+                        margin={['0', '0 auto']}
```

Comment:
can it be an object?

Reply:
I didn't change this, just auto formatted. 😂 Not part of the scope.